crates/rune/src/runtime/protocol.rs

```diff
@@ -81,6 +81,18 @@ impl Protocol {
         hash: Hash::new(0x418f5becbf885806),
     };
 
+    /// Search for a field by `String`
+    pub const DYNAMIC_FIELD_GET: Protocol = Protocol {
+        name: "dynamic_field_get",
+        hash: Hash::new(0x6dda58b140dfeaf9),
```

Comment:
Out of curiosity, how did you generate these?

Reply:
> Out of curiosity, how did you generate these?

I used the Inst hasher and used a name which wouldn't be allowed as a field name.